pages/admin.js

```diff
@@ -0,0 +1,116 @@
+import Head from "next/head";
+import db from "../data/database";
+import { products } from "../data/products";
+import { ListProductComponent } from "../components/product/list";
+import { useState, useEffect } from "react";
+
+let clearPage = () => {
```

Comment:
direct dom interaction is usually frowned upon in the react world. you can use the `useRef` to target an element in dom and interact with it which is better compatible with react.